build/azure-pipelines/updateGalleries.yml

```diff
@@ -0,0 +1,47 @@
+trigger:
+- release/extensions
+- extensions/rc1
+
+steps:
+- task: AzureFileCopy@4
+  displayName: 'Update extensionsGallery.json'
+  inputs:
+    SourcePath: extensionsGallery.json
+    azureSubscription: 'ClientToolsInfra_670062 (88d5392f-a34f-4769-b405-f597fc533613)'
```

Comment:
we should define this as a pipeline variable, right?

Reply:
We could but we already use this in other pipelines and it's highly unlikely to every change at this point so wasn't really something I saw being necessary